### PR TITLE
#172499309 Users can switch between multiple languages

### PR DESCRIPTION
**Pull request by bbaime98**
#### What does this PR do?
- add a switch between multiple languages feature 
#### Description of Task to be completed?
- install react-intl
- add i18n reducer and actions
- adjust navbar to switch languages when a certain language is selected
#### How should this be manually tested?
- Choose either English or French on the navigation bar or set English/French as your default browser language
#### What are the relevant pivotal tracker stories?
[#172499309](https://www.pivotaltracker.com/story/show/172499309)
#### Screenshots (if appropriate)
image
image

#### Questions:

### CODE REVIEW

File: src/__tests__/shared.js [hunk 1]
```diff
@@ -28,12 +28,29 @@ describe('Footer Tests', () => {
...
+  //   wrapper.setState({
+  //     language: '',
+  //   });
+  //   wrapper.find('[title]').simulate('select', '#en');
+  //   const { selectLanguage } = wrapper.instance().props;
+  //   selectLanguage.mockReturnValueOnce({ payload: 'en' });
+  //   wrapper.setState({
+  //     language: selectLanguage.mockReturnValueOnce({ payload: 'en' }),
+  //   });
+  // });
```
Comment (izzett222) [inline src/__tests__/shared.js]: can you remove the commented code

File: src/views/routes.js [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
+import sample from '../components/sharedComponents/notFound';
 import Home from '../components/sharedComponents/home';
 import LoginPage from '../components/authentication/Login';
 
 const Routes = () => (
   <Switch>
     <Route exact path="/" component={Home} />
     <Route exact path="/login" component={LoginPage} />
-    <Route component={NotFound} />
+    <Route component={sample} />
```
Comment (izzett222) [inline src/views/routes.js]: isn't the name NotFound more understandable than Sample on the not found component

**Review by izzett222 — CHANGES_REQUESTED** [review]

**Review by izzett222 — COMMENTED** [review]
Screen Shot 2020-05-11 at 12 13 26
could you add french for the copyright also

**Comment by Baraka-Mugisha** [comment]
Could you translate the input fields (`email and password`) too??

french

**Comment by bbaime98** [comment]
> Could you translate the input fields (`email and password`) too??
> 
> french

image

File: src/__tests__/app.test.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
+/* eslint-disable import/no-named-as-default */
+import * as React from 'react';
+// import { IntlProvider } from 'react-intl';
```
Comment (jabichris) [inline src/__tests__/app.test.js]: This is not good for the quality of your code. You should request a review when you are sure that everything is clean. Remove this if you don't need it.

File: src/__tests__/shared.js [hunk 4]
```diff
@@ -1,11 +1,11 @@
-
+/* eslint-disable import/named */
```
Comment (jabichris) [inline src/__tests__/shared.js]: We agreed that we are not going to disable other eslint rules. Fix this

File: src/components/App.js [hunk 5]
```diff
@@ -1,11 +1,35 @@
-import React from 'react';
+/* eslint-disable no-unused-vars */
+/* eslint-disable react/no-this-in-sfc */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable no-console */
+/* eslint-disable react/prop-types */
```
Comment (jabichris) [inline src/components/App.js]: And all the above rules should not be disabled

**Review by Ntare22 — APPROVED** [review]
LGTM

File: src/__mocks__/react-router-dom.js [hunk 6]
```diff
@@ -0,0 +1,7 @@
+/* eslint-disable react/prop-types */
```
Comment (jabichris) [inline src/__mocks__/react-router-dom.js]: Can you remove this so that we can merge this PR

**Pull request merged**